packages/ui/FloatingLayer/FloatingLayer.js

```diff
@@ -252,13 +257,15 @@ class FloatingLayerBase extends React.Component {
 		delete rest.onOpen;
 
 		if (open && this.state.nodeRendered) {
-			return ReactDOM.createPortal(
-				<div {...rest}>
+			const portal = ReactDOM.createPortal(
+				<div {...rest} id={id}>
 					{scrimType !== 'none' ? <Scrim type={scrimType} onClick={this.handleClick} /> : null}
 					{React.cloneElement(children, {onClick: this.stopPropagation})}
 				</div>,
 				this.node
 			);
+
+			return <div aria-owns={id}>{portal}</div>;
```

Comment:
Adding DOM here could be problematic. Since the portal is rendering in another subtree, this `<div />` ends up being dropped wherever `FloatingLayer` was used and those component might not be expecting it. 

I like the instinct to build this into `ui` but I'm not sure this is the right way. It might not be feasible to do since some outer component needs the `id` in order to set `aria-owns` correctly.

An alternative might be to add some `Context` to this to help auto-wire IDs up since they are required (instead of something generic like CSS classes) for ARIA to behave correctly. I'm not sure what that would be yet but something to consider.